packages/rancher-cis-benchmark/charts/values.yaml

```diff
@@ -4,8 +4,8 @@
 
 image:
   cisoperator:
-    repository: rancher/cis-operator
-    tag: v1.0.1
+    repository: prachidamle/cis-operator
+    tag: c1
```

Comment:
This is a placeholder - when the cis-operator PRs are merged, this will be updated